website/content/docs/concepts/tokens.mdx

```diff
@@ -287,3 +287,13 @@ to Vault).
 As a corollary, batch tokens can be used across performance replication
 clusters, but only if they are orphan, since non-orphan tokens will not be able
 to ensure the validity of the parent token.
+
+## Error Responses
+
+When using a token that has been revoked, exceeded it's TTL, or is an otherwise invalid value, Vault will respond
```

Comment:
```suggestion
When using a token that has been revoked, exceeded its TTL, or is an otherwise invalid value, Vault will respond
```

Reply:
"It's" means "it is", whereas "its" is the possessive (like "hers")

Reply:
oooops!!